Runes.podspec

```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |spec|
+  spec.name = 'Runes'
+  spec.version = '1.1.1'
+  spec.summary = 'Indecipherable symbols that some people claim have actual meaning.'
+  spec.homepage = 'https://github.com/thoughtbot/runes'
+  spec.license = { :type => 'MIT', :file => 'LICENSE' }
+  spec.author = {
+    'Gordon Fontenot' => 'gordon@thoughtbot.com',
+    'Adam Sharp' => 'adsharp@me.com',
```

Comment:
Remove Adam as an author (although I super duper appreciate his contribution as well as your effort in trying to track down the author info)